rclcpp/executors/cbg_executor/src/examples_rclcpp_cbg_executor/ping_node.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 Robert Bosch GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "examples_rclcpp_cbg_executor/ping_node.hpp"
+
+#include <algorithm>
+#include <chrono>
+#include <functional>
+#include <memory>
+
+#include <examples_rclcpp_cbg_executor/utilities.hpp>
```

Comment:
to stay consistent with the pong node
```suggestion
#include "./utilities.hpp"
```